### ISSUE TITLE
How to running this project with debug mode

### ISSUE THREAD

**Issue by zhang-zhen-project** [trainingl/STG4Traffic#3]
Dear author :
I was running 'STGCN_Main.py' with debug mode by vscode, and I found a issue 'No module named 'lib'', whether this project running with interminal mode only?

**Comment by trainingl** [comment]
Yes, we wrote this project in the server terminal (maybe it is a personal habit). If you want to debug the project, you can import sys in the project to modify the startup path to achieve this.

**Comment by trainingl** [comment]
> Dear author : I was running 'STGCN_Main.py' with debug mode by vscode, and I found a issue 'No module named 'lib'', whether this project running with interminal mode only?

Yes, we wrote this project in the server terminal (maybe it is a personal habit). If you want to debug the project, you can import sys in the project to modify the startup path to achieve this.
